src/Authenticator.ts

```diff
@@ -20,14 +21,23 @@ export type InfoTransformerFunction = (info: any) => Promise<any>
 export interface AuthenticatorOptions {
   key?: string
   userProperty?: string
+  sessionPlugin?: string
+}
+
+export interface SessionManagement {
```

Comment:
I think this should have the name `SessionManager` and the class should have the name `FastifySessionManager` or something like that?